doc/radosgw/config.rst

```diff
@@ -128,319 +130,509 @@ you have created all of the foregoing pools::
+portion of the Ceph configuration file as configuring a Ceph Storage Cluster
+client where the client type is a Ceph Object Gateway (i.e., ``radosgw``).
 
-Configuring Operations Logging
-------------------------------
 
-In early releases of Ceph (v0.66 and earlier), the Ceph Object Gateway will log
-every successful operation in the Ceph Object Gateway backend by default. This
-means that every request, whether it is a read request or a write request will
-generate a gateway operation that writes data. This does not come without cost,
-and may affect overall performance. Turning off logging completely can be done
-by adding the following config option to the Ceph configuration file::
+.. note:: The last line in the configuration i.e, ``rgw print continue = false``
+   is added to avoid issues with ``PUT`` operations.
 
-        rgw enable ops log = false
+Once you finish the setup procedure, if you encounter issues with your
+configuration, you can add debugging to the ``[global]`` section of your Ceph
+configuration file and restart the gateway to help troubleshoot any
+configuration issues. For example::
 
-Another way to reduce the logging load is to send operations logging data to a
-UNIX domain socket, instead of writing it to the Ceph Object Gateway backend::
+	[global]
+	#append the following in the global section.
+	debug ms = 1
+	debug rgw = 20
 
-        rgw ops log rados = false
-        rgw enable ops log = true
-        rgw ops log socket path = <path to socket>
 
-When specifying a UNIX domain socket, it is also possible to specify the maximum
-amount of memory that will be used to keep the data backlog::
+Distribute updated Ceph configuration file
+==========================================
 
-        rgw ops log data backlog = <size in bytes>
+The updated Ceph configuration file needs to be distributed to all Ceph cluster
+nodes from the ``admin node``.
 
-Any backlogged data in excess to the specified size will be lost, so the socket
-needs to be read constantly.
+It involves the following steps:
 
+#. Pull the updated ``ceph.conf`` from ``/etc/ceph/`` to the root directory of
+   the cluster in admin node (e.g. ``my-cluster`` directory). The contents of
+   ``ceph.conf`` in ``my-cluster`` will get overwritten. To do so, execute the
+   following::
 
-Enabling Subdomain S3 Calls
----------------------------
+		ceph-deploy --overwrite-conf config pull {hostname}
 
-To use a Ceph Object Gateway with subdomain S3 calls (e.g.,
-``http://bucketname.hostname``), you must add the Ceph Object Gateway DNS name
-under the ``[client.radosgw.gateway]`` section of your Ceph configuration file::
+   Here, ``{hostname}`` is the short hostname of the Ceph admin node.
 
-	[client.radosgw.gateway]
-		...
-		rgw dns name = {hostname}
+#. Push the updated ``ceph.conf`` file from the admin node to all other nodes in
+   the cluster including the ``gateway host``::
+
+		ceph-deploy --overwrite-conf config push [HOST][HOST...]
```

Comment:
Let's put spaces between each "`[HOST]`" entry here, so it's clearer that these are individual arguments to ceph-deploy.